charts/kueue/values.yaml

```
@@ -149,6 +149,8 @@ webhookService:
       protocol: TCP
       targetPort: 9443
   type: ClusterIP
+webhook:
```

```suggestion
mutatingWebhookConfiguration:
```

IIUC, ` reinvocationPolicy` is a mutating webhook configuration dedicated parameter. So, I think we would avoid using `webhook` so that we will be able to store the parameters used by both (validation / mutation).

@mimowo Any thoughts?

---

Indeed, the [reinvocationPolicy](https://github.com/kubernetes/kubernetes/blob/48fcf418ce08e0352b01a7830d97e12b62f80f27/staging/src/k8s.io/api/admissionregistration/v1/types.go#L1079) field is specific to mutating webhooks, inside the MutatatingWebhook struct.

So, maybe shorted `mutatingWebhook` is enough, wdyt @tenzen-y ?

---

> IIUC, ` reinvocationPolicy` is a mutating webhook configuration dedicated parameter. So, I think we would avoid using `webhook` so that we will be able to store the parameters used by both (validation / mutation).
> 
> @mimowo Any thoughts?

totally make sense, fixed :)

---

> So, maybe shorted mutatingWebhook is enough, wdyt @tenzen-y ?

SGTM